package.json

```diff
@@ -1,18 +1,20 @@
 {
   "name": "eslint-config-videoamp",
   "version": "4.2.0",
```

Comment:
We'll need to bump this up to a major version

Reply:
Yes, was planning on that